modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GcalControllerEpics.scala

```diff
@@ -22,6 +22,7 @@ object GcalControllerEpics extends GcalController {
       v match {
         case BinaryOnOff.OFF => LampState.Off
         case BinaryOnOff.ON  => LampState.On
+        case _               => sys.error("Cannot happen")
```

Comment:
@tpolecat This one surprised me. `BinaryOnOff` is a Java enumeration with only two values, however the typelevel compiler puts a warning that the match is not fully exhaustive. The fix above makes it work but IMHO it shouldn't be necessary